src/main/java/seedu/address/model/Pet.java

```diff
@@ -28,6 +28,18 @@ public void setName(String name) {
         this.name = name;
     }
 
+    // When a task is done, exp level increases by 5
```

Comment:
so I'm assuming your pet implements the model interface?

Reply:
No it does not. Pet object is just an object that represents the pet!